src/index.js

```diff
@@ -0,0 +1,180 @@
+      },
+    })
+    .then((response) => {
+      let lat = response.data[0].lat;
+      let lon = response.data[0].lon;
+
+      return {
+        latitude: lat,
+        longitude: lon,
+      }
+    })
+    .catch((error) => {
+      console.log('Unknown latitude and longitude:', error.response);
+    });
+};
+
+const getWeather = (location) => {
+  return axios
+    .get('http://localhost:5000/weather', {
+      params: {
+        lat: location.latitude,
+        lon: location.longitude,
+      },
+    })
+    .then((response) => {
+      let targetTemp = response.data.main.temp
+      return Math.round(convertKtoF(targetTemp));
+    })
+    .catch((error) => {
+      console.log('Error getting the weather:', error);
+    });
+  };
+  
+  const applyColorAndGarden = () => {
+    state.tempValue.textContent = state.tempInt;
+    
+    let landscape = '';
+    if (state.tempInt < 0 || state.tempInt > 120) {
+      state.tempValue.style.color = 'red';
+      state.tempValue.textContent = 'DANGER'
+      landscape = '🚫🚫🚫TOO MUCH WEATHER🚫🚫🚫'; 
+    } else if (state.tempInt>= 80) {
+      state.tempValue.style.color = 'red';
+      landscape = '🌵🏜️🌵🐍🦂🌋👹🌋🦂🐍🌵🏜️🌵';
+    } else if (state.tempInt>= 70) {
+      state.tempValue.style.color = 'orange';
+      landscape = '🌞🌞🌻🌻🏖️🏝️👙🏝️🏖️🌻🌻🌞🌞';
+    } else if (state.tempInt>= 60) {
+      state.tempValue.style.color = 'gold';
+      landscape = '🌹🌸🎋🌹🌸🌱🪺🌱🌸🌹🎋🌸🌹';
+    } else if (state.tempInt>= 50) {
+      state.tempValue.style.color = 'green';
+      landscape = '🌳🌾🍂🪵🌳🐿️🧣🐿️🌳🪵🍂🌾🌳';
+    } else if (state.tempInt>= 40) {
+      state.tempValue.style.color = 'teal';
+      landscape = '🌲🌳🌨️🪵🌲🌳🧤🌳🌲🪵🌨️🌳🌲';
+    } else if (state.tempInt< 40) {
+      state.tempValue.style.color = 'aqua';
+      landscape = '🎄❄️🌨️🌲⛄️🌨️❄️🌨️⛄️🌲🌨️❄️🎄'; 
+    }
+    
+    state.landscapeElement.textContent = landscape;
+  };
+  
+  const increaseTemp = () => {
+    state.tempInt++;
+    applyColorAndGarden();
+  };
+  
+  const decreaseTemp = () => {
+    state.tempInt--;
+    applyColorAndGarden();
+  };
+  
+  const updateCity = () => {
+    state.city = state.cityNameInput.value;
+    state.cityHeader.textContent = state.city;
+  }
+
+  
+  const skyView = (event) => {
+    let skyColor = event.target.value;
+
+  
+    if (skyColor === 'Cloudy') {
+      state.sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+      state.gardenSection.setAttribute('class', 'garden__section cloudy')
+      }else if (skyColor === 'Sunny') {
+      state.sky.textContent = '☁️   🌞☀️  🌤️  ☁️ ☀️ ☁️  ☁️';
+      state.gardenSection.setAttribute('class', 'garden__section sunny');
+      }else if (skyColor === 'Rainy') {
+      state.sky.textContent = '🌧🌈☔️🌧🌧💧🌧🌦🌧💧☔️🌧🌧';
+      state.gardenSection.setAttribute('class', 'garden__section rainy')
+      }else if (skyColor === 'Snowy') {
+      state.sky.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+      state.gardenSection.setAttribute('class', 'garden__section snowy');
+    }
+  };
+  
+
+  
```

Comment:
Small nitpitck: we should review our code to ensure we're consistently applying spacing between functions across the project to avoid confusion from readers/maintainers around if functions are grouped for a reason or if something was added or removed and the spacing wasn't updated.